src/style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  outline: none;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+form {
+  margin: 20px 0;
+}
+
+.big--input {
+  background-repeat: no-repeat;
+  background-position: 26px;
+  background-image: url(./images/Search.svg);
+  padding-left: 62px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
+  color: #3d4e61;
```

Comment:
Move all repeatable colors to the variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)